core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -246,6 +247,14 @@ def install_routes
       end
     end
 
+    def install_solidus_starter_frontend
+      return unless install_solidus_starter_frontend?
+
+      say_status :install, "Solidus Starter Frontend"
+      ENV['LOCATION'] ||= 'https://raw.githubusercontent.com/nebulab/solidus_starter_frontend/master/template.rb'
+      run 'bin/rails app:template'
```

Comment:
We can add an option to this generator to skip installing the frontend specs test suite, something like `with_frontend_specs`. We can pass the value of this option to `run 'bin/rails app:template'` via an environment variable `ENV['SKIP_SPECS']`. See https://github.com/nebulab/solidus_starter_frontend/pull/185/files#diff-1ceb42e9dd4ccfd7b01826cb5e570004ec897791b4f43d0117d2a6e4d5717ea5R139. Let me know what you think!